app/main.py

```diff
@@ -224,4 +224,4 @@ def reset_all():
 if __name__ == "__main__":
     import uvicorn
 
-    uvicorn.run("main:app", host="0.0.0.0", port=7865, reload=True)
+    uvicorn.run("main:app", host="0.0.0.0", port=7865, reload=False)
```

Comment:
Setting reload to False is a bit painful to development, but we can sort this over time :)